datasets/n2c2_datasets/synthetic_n2c2_datasets/create_synthetic_note.py

```diff
@@ -84,4 +80,10 @@ def create_synthetic_note(sent_add, words_replace):
                         default=50)
     # parsing arguments
     args = parser.parse_args()
-    create_synthetic_note(args.s_add, args.w_replace)
+    file_name = '/Users/alissavalentine/Charney_rotation/redundancy_transformers/datasets/n2c2_datasets/' \
+                'synthetic_n2c2_datasets/train_sentences.txt'
+    output_notes = "synthetic_notes.txt"
+    output_word_metadata = "word_metadata.txt"
+    output_sent_metadata = "sentence_metadata.txt"
+    create_synthetic_note(args.s_add, args.w_replace, file_name)
```

Comment:
Add `file_name`, `output_notes`, `output_word_metadata`, `output_sent_metadata` to the parser, you should be modifying your folder/file names only in the bash script.

`create_synthetic_note` takes as input 6 objects, see previous comment.

Reply:
got it